bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/handler/PulseaudioHandler.java

```diff
@@ -248,11 +327,39 @@ public void onDeviceStateChanged(ThingUID bridge, AbstractAudioDeviceConfig devi
         }
     }
 
+    @SuppressWarnings("null")
+    public String getHost() {
+        if (getBridge() != null) {
+            return (String) getBridge().getConfiguration().get(PulseaudioBindingConstants.BRIDGE_PARAMETER_HOST);
```

Comment:
You could store the result of `getBridge()` to a local variable and do the null check on that to get rid of the suppression.

Reply:
Oh damnit, I let this one pass through !